cli/cmd/resources/odiglet.go

```diff
@@ -411,6 +445,45 @@ func NewOdigletDaemonSet(ns string, version string, imagePrefix string, imageNam
 		},
 	}
 }
+
+// used to inject the host volumes into odigos components for selinux update
+func selinuxHostVolumes() []corev1.Volume {
+	return []corev1.Volume{
+		{
+			Name: "host",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/",
```

Comment:
Can we mount `/sbin` instead of `/`?

Reply:
that is what I tried first, but semanage is a python script and python is installed outside of /sbin (and I am sure if we want to manage all the python dependencies in the odiglet container), other then making this readonly, I am not too happy about mounting it... what else can you think of?